Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+    <PropertyGroup>
+        <PackageProjectUrl>https://zaid-ajaj.github.io/Feliz/</PackageProjectUrl>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="DotNet.ReproducibleBuilds" Version="1.1.1" PrivateAssets="all" />
```

Comment:
this package reference enables sourcelink automatically, as well as setting a few project properties like the RepositoryUrl.